application/helper/ApplicationUtils.php

```diff
@@ -238,7 +238,7 @@ public static function checkResourcePermissions(ConfigManager $conf, bool $minim
                  $conf->get('resource.update_check'),
              ] as $path
         ) {
-            if (!is_file(realpath($path))) {
+            if (!is_file(realpath($path ?? ''))) {
```

Comment:
Same result but it lifts any ambiguity
```suggestion
            if (!is_string($path) || !is_file(realpath($path))) {
```